specification/connector/list/ConnectorListRequest.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { RequestBase } from '@_types/Base'
+import { Fields } from '@_types/common'
+import { integer } from '@_types/Numeric'
+
+/**
+ * Returns existing connectors.
+ * @rest_spec_name connector.list
+ * @availability stack since=8.12.0 stability=experimental
+ * @availability serverless stability=experimental visibility=public
+ * @doc_id connector-list
+ */
+export interface Request extends RequestBase {
+  query_parameters: {
+    /**
+     * Starting offset (default: 0)
+     */
+    from?: integer
+    /**
+     * Specifies a max number of results to get
+     */
+    size?: integer
+    /**
+     * A comma-separated list of connector index names to fetch connector documents for
+     */
+    index_name?: Fields
```

Comment:
`Fields` is meant to hold a list of index field names. If these are ES indices, then `Indices` should be used.